server/tests/rooms/game/index.test.ts

```diff
@@ -199,4 +201,18 @@ describe('inverting pending inventory', () => {
     expect(g.players['Curator'].pendingInvestments.culture).toBe(-3);
     expect(g.players['Curator'].inventory.culture).toBe(3);
   });
-});
\ No newline at end of file
+});
+
+describe('out of commission event', () => {
+  const g = new GameState(mockGameStateInitOpts(x => x, () => 10));
+  const outOfCommissionEntreprenuer = getAllMarsEvents()[7][0];
```

Comment:
this is not going to work if the array of MarsEvents gets new things or changes its order, etc. Which it certainly will.

Please change this to pull the out of commission entrepreneur MarsEvent by a more reliable method, even if you have to "inefficiently" filter `id='outOfCommissionEntrepreneur'` instead. (as bonus points, we'll see if we are speling Entrepreneur correctly too :grin:)